modules/installation-osp-kuryr-octavia-configuration.adoc

```diff
@@ -116,13 +116,20 @@ backend is ML2/OVS. There is no need for modifications if the backend is
 ML2/OVN.
 ====
 
-. To enforce network policies across Services, like when traffic goes through
+. (Optional, not needed for Red Hat OpenStack Platform 16+) To enforce
+network policies across Services, like when traffic goes through
 the Octavia load balancer, you must ensure Octavia creates the Amphora VM
 security groups on the user project. To do that, you must add the project ID
 to the `octavia.conf` configuration file after you create the project.
 +
 This ensures that required LoadBalancer security groups belong to that project
 and that they can be updated to enforce Services isolation.
++
+[NOTE]
+====
+This is not needed for Red Hat OpenStack Platform 16+ as Octavia implements a
```

Comment:
"You do not need to ensure that Octavia creates the security groups in {rh-openstack} version 16 or later."

Reply:
I think the current text gives more information that the proposed change

Reply:
I'd probably just keep my version and link out to the RHOSP release notes if the why is necessary, but I'm okay with this.